nikola/conf.py.in

```diff
@@ -243,6 +243,12 @@ POSTS_SECTIONS = True
 # are the default and will apply GENERATE_ATOM if set.
 # POSTS_SECTIONS_ARE_INDEXES = True
 
+# Final locations are:
+# output / TRANSLATION[lang] / SECTION_PATH / SECTION_NAME / index.html (list of posts for a section)
+# output / TRANSLATION[lang] / SECTION_PATH / rss.xml (RSS feed for a section)
```

Comment:
That doesn’t make much sense. Did you mean `SECTION_PATH / SECTION_NAME / rss.xml`? (Otherwise, there would be only one RSS file for all sections)